browser_use/agent/service.py

```diff
@@ -238,6 +245,20 @@ def _set_message_context(self) -> str | None:
 				self.settings.message_context = f'Available actions: {self.available_actions}'
 		return self.settings.message_context
 
+	def _check_env_variables(self) -> bool:
```

Comment:
```suggestion
def check_env_variables(keys: list[str], any_or_all=all) -> bool:
    """top-level helper function to check env var values"""
    return any_or_all(os.getenv(key).strip() for key in keys)
```

- can be a pure util function so we can import it elsewhere too (e.g. from the examples)
- then at startup we can call and check for the keys needed for that run based on config e.g. `check_env_variables(['OPENAI_API_KEY', 'CHROME_USER_DATA_DIR'])`

Reply:
Makes sense

Reply:
yes, moving it to `utils.py` is what I had in mind 👍